build/generate-example.js

```diff
@@ -42,11 +41,20 @@ export default function generateExample({skipBuild} = {}) {
     sh.cp(path.join(vjsSwf, 'dist', 'video-js.swf'), swfDest);
   }
 
-  const files = klawSync('sandbox/').filter((file) => path.extname(file.path) === '.example');
+  const filepaths = sh.find(path.join(__dirname, '..', 'sandbox', '**', '*.*'))
```

Comment:
klawSync is more readable, I think

Reply:
We could use just `sandbox/` for `find` too, but then the scripts don't work if run from somewhere other than the root dir